raiden/tests/integration/rpc/assumptions/test_rpc_gas_price_assumptions.py

```diff
@@ -238,8 +238,8 @@ def test_remote_transaction_with_zero_gasprice_is_not_mined(
     except TransactionNotFound:
         zerogas_receipt = None
 
-    msg = "The transaction does NOT have a receipt because the miner rejected it"
-    assert zerogas_receipt is None, msg
+    msg = "The transaction receipt does NOT have a blockHash because the miner rejected it"
+    assert zerogas_receipt["blockHash"] is None, msg
```

Comment:
```suggestion
    assert zerogas_receipt and zerogas_receipt["blockHash"] is None, msg
```

Reply:
Yes, that's what you get for testing with only one eth client locally.